app/models/likes

```diff
@@ -0,0 +1,13 @@
+class Like < ApplicationRecord
+    belongs_to :user
+    belongs_to :post
+
+    after_save :update_post_likes_count
+
+    private
+
+    def update_post_likes_count
+        post_likes_count = Comment.where(post_id: post_id).count
+        post.update(likesCounter: post_likes_count)

```

Comment:
:red_circle: [REQUIRED] Please ensure that you adhere to Ruby's best practices for variable naming. The naming convention for variables is` snake_case` for methods and variables. So `likesCounter` should be `likes_counter`